src/services/user.js

```diff
@@ -7,10 +7,18 @@ import http from './http';
 import session from './session';
 import storage from './storage';
 
+/**
+ * @global
+ * @typedef User
```

Comment:
Since we already have a global typedef called `User`, we should rename this one to something like `CLWCredits` or `ChapelCredits`.